frontend/public/components/software-details.jsx

```diff
@@ -44,12 +44,12 @@ export const SubHeaderRow = ({header, children}) => {
   </div>;
 };
 
-const SoftwareDetailRow = ({title, detail, text, children}) => {
-  return <div className="row cluster-overview-cell__info-row">
-    <div className="col-xs-6">
+const SoftwareDetailRow = ({className, title, detail, text, children, noColumns, titleClassName, detailClassName}) => {
```

Comment:
I'm not sure we should reuse this component. We've talked about removing it, so I might just move the two requests to get the versions into the about modal.